### DISCUSSION TITLE
External Link in NavBar

### DISCUSSION THREAD

**Discussion by AstroAnmol** [alshedivat/al-folio#579] [Q&A] [answered]
I have a blog on [substack](https://blog.aerospacenerd.com/) and would like to link that in the navbar. I am pretty new to Jekyll, can someone help me?

**Comment by alshedivat** [comment, marked as answer]
you should be able to add arbitrary links to the navbar by editing `_includes/header.html` and adding a few lines of code similar to the one that adds the about page link:
https://github.com/alshedivat/al-folio/blob/55ca811a9ab20c1c6c99b00952fcdb3587b1f0a3/_includes/header.html#L28-L35
for your substack link, you can have something like this:
```html
<!-- Substack --> 
<li class="nav-item"><a class="nav-link" href="https://blog.aerospacenerd.com/">substack</a></li> 
```
